### PR TITLE
[doc] PR for Document upgrade an Airgapped YBA

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6268

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+linkTitle: Upgrade air-gapped installation
+description: Upgrade air-gapped YugabyteDB Anywhere installation
+menu:
+  preview_yugabyte-platform:
+    identifier: upgrade-yp-airgapped
+    parent: upgrade
+    weight: 81
+type: docs
+---
+
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md:13]: Should these pages be on tabs like https://deploy-preview-15077--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/default/

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+You can upgrade your air-gapped installation of YugabyteDB Anywhere to a newer version as follows:
+
+1. Manually obtain and move the binary Replicated license file `<filename>.rli` to the `/home/{username}/`  directory. 
+
+2. Manually obtain and move the YugabyteDB Anywhere air-gapped package to the `/opt/yugabyte/releases/<new_version_dir>` directory. For example, if you are upgrading to the YugabyteDB Anywhere version 2.14, you would perform the following: 
+
+   Execute the following command to obtain the package:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md:20]: bullet these 2 substeps?

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+   Create the `/opt/yugabyte/releases/yugaware-2.14.0.0-b94/` directory and move (or SCP) the `yugaware-2.14.0.0-b94-linux-x86_64.airgap` file into that directory.
+
+3. Log in to the Replicated Admin Console at <https://:8800/> and navigate to **Settings** to load the new license file, as per the following illustration: 
+
+   ![img](/images/yp/airgap-settings.png)
+
+   Change the two directories to match the ones you used. For example, enter `/opt/yugabyte/releases/yugaware-2.14.0.0-b94/` in the **Update Path** field and `/home/{user}/` in the **License File** field.
+
+   Replicated will detect updates based on the updated path information and will apply them in the same way it does for connected YugabyteDB Anywhere installations.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md:34]: Replicated detects updates based on the updated path information and applies them in the same way it does for connected YugabyteDB Anywhere installations.

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md:13]: I'm not sure. I'm planning to create a separate ticket for the tabs and look at every section.

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md:20]: I rephrased it.

**Pull request merged**